dpuagent/ansible/dpuagent.py

```
@@ -0,0 +1,130 @@
+zstacklib_args.require_python_env = require_python_env
+zstacklib_args.zstack_releasever = releasever
+if host_info.distro in DEB_BASED_OS:
+    zstacklib_args.apt_server = yum_server
+    zstacklib_args.zstack_apt_source = zstack_repo
+else :
+    zstacklib_args.yum_server = yum_server
+zstacklib = ZstackLib(zstacklib_args)
+
+# name: copy dpuagent binary
+command = 'rm -rf {};mkdir -p {}'.format(dpuagent_root, dpuagent_root + "/certs")
+run_remote_command(command, host_post_info)
```

Ruff (0.15.2)

[error] `run_remote_command` may be undefined, or defined from star imports

(F405)